platform/darwin/src/MGLFillExtrusionStyleLayer.h

```diff
@@ -0,0 +1,107 @@
+// This file is generated. 
+// Edit platform/darwin/scripts/generate-style-code.js, then run `make style-code-darwin`.
+
+#import "MGLStyleValue.h"
+#import "MGLVectorStyleLayer.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ Controls the translation reference point.
+ 
+ Values of this type are used in the `fillExtrusionTranslateAnchor` property of `MGLFillExtrusionStyleLayer`.
+ */
+typedef NS_ENUM(NSUInteger, MGLFillExtrusionTranslateAnchor) {
+    /**
+     The fill extrusion is translated relative to the map.
+     */
+    MGLFillExtrusionTranslateAnchorMap,
+    /**
+     The fill extrusion is translated relative to the viewport.
+     */
+    MGLFillExtrusionTranslateAnchorViewport,
+};
+
+/**
+ A fill-extrusion layer which allows customization of styling properties at runtime. You may 
+ instantiate a new fill-extrusion layer to add to a map style or you may query an 
+ `MGLMapView` for its `style` and obtain existing layers using the 
+ `-[MGLStyle layerWithIdentifier:]` method. 
+ */
+@interface MGLFillExtrusionStyleLayer : MGLVectorStyleLayer
+
+#pragma mark - Accessing the Paint Attributes
+
+/**
+ The height with which to extrude the base of this layer. Must be less than or equal to `fillExtrusionHeight`.
+
+ This property is measured in meters.
+ 
+ The default value of this property is an `MGLStyleValue` object containing an `NSNumber` object containing the float `0`. Set this property to `nil` to reset it to the default value.
+
+ This property is only applied to the style if `fillExtrusionHeight` is non-`nil`. Otherwise, it is ignored.
+ */
+@property (nonatomic, null_resettable) MGLStyleValue<NSNumber *> *fillExtrusionBase;
+
+#if TARGET_OS_IPHONE
+/**
+ The base color of the extruded fill. The extrusion's surfaces will be shaded differently based on this color in combination with the root `light` settings. If this color is specified as `rgba` with an alpha component, the alpha component will be ignored; use `fillExtrusionOpacity` to set layer opacity.
+ 
+ The default value of this property is an `MGLStyleValue` object containing `UIColor.blackColor`. Set this property to `nil` to reset it to the default value.
+
+ This property is only applied to the style if `fillExtrusionPattern` is set to `nil`. Otherwise, it is ignored.
+ */
+@property (nonatomic, null_resettable) MGLStyleValue<MGLColor *> *fillExtrusionColor;
+#else
+/**
+ The base color of the extruded fill. The extrusion's surfaces will be shaded differently based on this color in combination with the root `light` settings. If this color is specified as `rgba` with an alpha component, the alpha component will be ignored; use `fillExtrusionOpacity` to set layer opacity.
```

Comment:
Per https://github.com/mapbox/mapbox-gl-native/pull/6107#issuecomment-257907176, we override documentation to remove the phrase “as `rgba`”, because this is CSS syntax, and alpha may be specified with `-colorWithHue:saturation:brightness:alpha:` or `-colorWithWhite:alpha:` instead of `-colorWithRed:green:blue:alpha:`.